packages/did-auth-siop-op-authenticator/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@sphereon/ssi-sdk-did-auth-siop-authenticator",
+  "version": "0.0.1",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "veramo": {
+    "pluginInterfaces": {
+      "IDidAuthSiopOpAuthenticator": "./src/types/IDidAuthSiopOpAuthenticator.ts"
+    }
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "generate-plugin-schema": "yarn veramo dev generate-plugin-schema"
+  },
+  "dependencies": {
+    "@sphereon/did-auth-siop": "^0.2.2",
+    "@sphereon/pe-js": "^0.5.0",
```

Comment:
0.5.1 of PE-JS is released with some critical fixes we need for this integration to actually start working in creating VPs in production scenarios later on

Reply:
did-auth-siop is not yet compatible with pe-js 0.5.1

Reply:
A new version 0.2.3 is just released

Reply:
implemented